pkgs/tools/misc/kamp/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  replaceVars,
+  perl,
+  stdenv,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "kamp";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "vbauerster";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-9cakFhA9niMZ0jD0ilgCUztk4uL6wDp6zfHUJY/yLYw=";
+  };
+
+  cargoHash = "sha256-BnVV0UnXEebq1kbQvv8PkmntLK0BwrOcMxxIODpZrxc=";
+
+  postInstall = ''
+    install scripts/* -Dt $out/bin
+  '';
+
+  meta = {
```

Comment:
Add supported `platforms` - this might be `lib.platforms.linux`!

Reply:
I don't see why this would be Linux-only. The default of platforms.all seems sensible here.

Reply:
Great call - I did not give a good suggestion here. Mentioning @erikeah here since the thread might be hidden.